FakebookNotifications/FakebookNotifications.Testing/Repo Tests/NotificationRepoTests.cs

```diff
@@ -8,16 +8,21 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Xunit;
+using Microsoft.Extensions.Logging;
 
 namespace FakebookNotifications.Testing
 {
     public class NotificationRepoTests
     {
         private Mock<IOptions<NotificationsDatabaseSettings>> _mockSettings;
         private NotificationsDatabaseSettings settings;
+        private readonly ILogger<NotificationsContext> _logger;
 
-        public NotificationRepoTests()
+        public NotificationRepoTests(ILogger<NotificationsContext> logger)
```

Comment:
there isn't a dependency injector in xunit. just mock the logger, or instantiate a `NullLogger<NotificationsContext>`.